workspaces/liferay-partner-workspace/quickstart/docker-compose.yaml

```diff
@@ -27,5 +27,6 @@ services:
         image: liferay/dxp:2024.q2.6
         ports:
             - 8080:8080
```

Comment:
@abfranca can you sort these to make the 8000 above 8080? other than that looks good!